modules/luigi/phrases/farewell.yml

```diff
@@ -0,0 +1,20 @@
+type: phrases
+name: "payment"
+opts:
+- "alrighty, payment time then! you owe me {cost} roboy coins for the {order}"
+- "Wow it took some time. That will be {cost} roboy coins for the {order} "
+- "Do you want some extra fries with that? Just kidding {cost} roboy coins for the {order}"
+---
+type: phrases
+name: "continue_order"
+opts:
+- "wow, you are quite hungry. then tell me what other ice cream you want!"
+- "It is never enough! Tell me what other ice cream you want "
+- "That's the spirit ! What more do you want?"
```

Comment:
Lower case again :) and possibly remove the unnecessary space in front of the `!`? 😉